modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -69,6 +73,7 @@ export default {
         data: this.next,
         error: this.validate(this.next)
       });
+      apos.bus.$emit(this.field.name, this.next);
```

Comment:
The slug field needs to know the value of a sibling (usually `title`).

I did that to avoid to pass to every component in `AposSchema` all fields data. idk if it's the best move.

Reply:
This is a lot of data flying through the bus. At the very least, the event name here should be more specific, such as `field-update-${this.field.name}`, though you'd probably want the doc ID in there to prevent issues with nested editor modals (in cases of relationship editing, for example).

I'm wondering if we should hard-code this special case into AposSchema. It's unique but also extremely common. There are many ways to do this.

Or `AposSchema` looks for the `slugifies` field property and handles business logic based on that?

Or maybe we use the bus, but include the doc ID. It does seem like using the bus is excessive when ultimate we're only working one level up and down.